core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -38,6 +38,7 @@ registry.ci.openshift.org/origin/4.11:baremetal-runtimecfg quay.io/openshift/ori
 registry.ci.openshift.org/origin/4.11:base quay.io/openshift/origin-base:4.11 quay.io/openshift/origin-base:4.11.0 quay.io/openshift/origin-base:latest
 registry.ci.openshift.org/origin/4.11:build-test-qemu-img quay.io/openshift/origin-build-test-qemu-img:4.11 quay.io/openshift/origin-build-test-qemu-img:4.11.0 quay.io/openshift/origin-build-test-qemu-img:latest
 registry.ci.openshift.org/origin/4.11:cert-manager-operator quay.io/openshift/origin-cert-manager-operator:4.11 quay.io/openshift/origin-cert-manager-operator:4.11.0 quay.io/openshift/origin-cert-manager-operator:latest
+registry.ci.openshift.org/origin/4.11:ci-image quay.io/openshift/origin-ci-image:4.11 quay.io/openshift/origin-ci-image:4.11.0 quay.io/openshift/origin-ci-image:latest
```

Comment:
You do not want to promote this image, you should use `excluded_images` and then the tools will not require changing this file (https://github.com/openshift/release/pull/26716/files#diff-b0a8f6feb1bde5878f221db73719e280f35a896e2f5404dce4dcecee7e3fef7eR33-R34 )

Reply:
Thank you! I feel stupid coz I actually had it in my changes originally, but I made a mistake committing which meant I had to re-do it and lost that bit. But ass-you-me'd I had redone it correctly. Thank you and the review process =D